lnbits/core/static/js/service-worker.js-e

```diff
@@ -0,0 +1,58 @@
+// update cache version every time there is a new deployment
```

Comment:
lnbits/core/static/js/service-worker.js-e ? is this a error?

Reply:
- `make bundle` generated it for me
- I have removed it